uiowa/uids
# Feature: UIDS SiteNow Preparations

# How to test

Resolves #117 
 - This can only be verified by testing with https://github.com/uiowa/uiowa/pull/1334.

Resolves #116 
 - Go to http://localhost:3000/components/detail/events-block and add very long location text and verify that icon is positioned absolutely. 
 - Go to http://localhost:3000/components/detail/events-block and verify that the events block looks close to what was demoed in this morning's design meeting. 

<img width="1507" alt="Screen Shot 2020-04-08 at 2 51 13 PM" src="https://user-images.githubusercontent.com/1036433/78827254-6ab5fe00-79a8-11ea-9caa-a5e4992c5b27.png">
<img width="1336" alt="Screen Shot 2020-04-08 at 2 51 05 PM" src="https://user-images.githubusercontent.com/1036433/78827266-6e498500-79a8-11ea-8645-cb6c04f03294.png">


Resolves #115 
 - Collapse http://localhost:3000/components/detail/uiowa down and verify that card body is still visible. 


Resolves #114 
 - Remove `grid--3-2-stacked` class from http://localhost:3000/components/detail/stat-block and verify that collapse to mobile pattern goes to two columns around 670px instead of 1300px. 

Resolves #113 
 - Go to http://localhost:3000/components/detail/uiowa and verify that the underline on "Hear from Hawkeyes" has padding below it and is visible. 

Card bttn focus/hover state
 - Go to http://localhost:3000/components/preview/card-block--default and hover over a card and verify that an underline animation pops up from the bottom of the button.
